dat/outfits/structure/nanofiber_structures.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Nanofiber Structures">
```

Comment:
I'm not entirely sold on the name. I would consider also making the outfit much stronger, but make it so that if used on a large slot it gives 50% bonus only. You can do that by using the Lua API, but I can help with that later if necessary.